integration/src/main/java/scripts/api/SnapshotsApi.java

```diff
@@ -0,0 +1,58 @@
+package scripts.api;
+
+import bio.terra.datarepo.client.ApiException;
+import bio.terra.datarepo.model.DatasetModel;
+import bio.terra.datarepo.model.PolicyMemberRequest;
+import bio.terra.datarepo.model.SnapshotRequestContentsModel;
+import bio.terra.datarepo.model.SnapshotRequestModel;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.client.DatarepoClient;
+
+public class SnapshotsApi {
```

Comment:
why not extend snapshotsapi anymore?

Reply:
The same reason as not extending `DatasetsApi`; since in general I think it's better to encapsulate and so the caller doesn't try to call `deleteSnapshot()` directly. If it turned out that there were many APIs that needed forwarding I may have left it as-is to avoid all the forwarding APIs. But in this case there were only a few APIs that were forwarded.